src/pages/theming/dark-mode.md

```diff
@@ -0,0 +1,121 @@
+
+## Using Media Queries
+
+The first method available is the CSS media query for the user's preferred color scheme. This media query will hook into the system setting of the user's device and apply the changes if a dark mode is enabled.
+
+The fist values that will need to change are the text color and background color. As the name "dark mode" implies, the background of an app should be dark, with lighter text.
+
+```css
+@media (prefers-color-scheme: dark) {
+  :root {
+    --ion-background-color: var(--ion-color-dark);
+    --ion-background-color-rgb: var(--ion-color-dark-rgb);
+
+    --ion-text-color: var(--ion-color-light);
+    --ion-text-color-rgb: var(--ion-color-light-rgb);
+  }
+}
+```
+
+These will take the value from the [default Ionic colors](https://ionicframework.com/docs/theming/advanced#colors) in an app. Depending on any other customizations that have been made to the default theme, some adjustments may be needed. The overall idea is that the dark mode that is applied uses the same colors in an app's theme and creates a cohesive UI.
+
+In addition, the step variables can also be modified to further enhance the dark mode experience.
+
+In this case, the color steps are generated based on the background color and text color. The [stepped color generator](/docs/theming/advanced#generate-stepped-color-variables) can take the hex value of the background color and text color and automatically generate the steps needed for an app.
+
+```css
+@media (prefers-color-scheme: dark) {
+  :root {
+    --ion-background-color: var(--ion-color-dark);
+    --ion-background-color-rgb: var(--ion-color-dark-rgb);
+
+    --ion-text-color: var(--ion-color-light);
+    --ion-text-color-rgb: var(--ion-color-light-rgb);
+
+    --ion-color-step-50: #2d2e32;
+    --ion-color-step-100: #37393d;
+    --ion-color-step-150: #424347;
+    --ion-color-step-200: #4c4e52;
+    --ion-color-step-250: #57585c;
+    --ion-color-step-300: #616366;
+    --ion-color-step-350: #6c6d71;
+    --ion-color-step-400: #76787b;
+    --ion-color-step-450: #818286;
+    --ion-color-step-500: #8b8d90;
+    --ion-color-step-550: #96979a;
+    --ion-color-step-600: #a0a1a5;
+    --ion-color-step-650: #abacaf;
+    --ion-color-step-700: #b5b6ba;
+    --ion-color-step-750: #c0c1c4;
+    --ion-color-step-800: #cacbce;
+    --ion-color-step-850: #d5d6d9;
+    --ion-color-step-900: #dfe0e3;
+    --ion-color-step-950: #e9ebee;
+  }
+}
+```
+
+## Class Fallback
+
+Currently, the `prefers-color-scheme` media query is only supported in desktop Safari and mobile Firebox, so users will not able to benefit from having the dark mode applied automatically in other browsers. However, the dark mode can still be applied by styling a CSS selector and applying the class to our document body
```

Comment:
```suggestion
Currently, the `prefers-color-scheme` media query has [limited browser support](https://caniuse.com/#feat=prefers-color-scheme), so users will not be able to benefit from having the dark mode applied using this media query in certain browsers. However, the dark mode can still be applied by styling a CSS selector and applying the class to our document body
```

Maybe here we should just link to the browser support? That way we don't have to update the docs as support grows.